gis/grass/Portfile

```
@@ -1,11 +1,300 @@
+    }
+}
+
+variant mysql5 description {Add MySQL 5 support} {
+    depends_lib-append      path:bin/mysql_config5:mysql5
+    configure.args-append   --with-mysql
+    configure.args-append   --with-mysql-includes=${prefix}/include/mysql5/mysql
+    configure.args-append   --with-mysql-libs=${prefix}/lib/mysql5/mysql
+    configure.env-append    MYSQLD_CONFIG=${prefix}/lib/mysql5/bin/mysql_config
+}
+
+# PostGreSQL variants (from the GDAL port)
+set postgresql_suffixes {10 11 12 13}
+
+set postgresql_variants {}
+foreach suffix ${postgresql_suffixes} {
+    lappend postgresql_variants postgresql${suffix}
+}
+
+foreach suffix ${postgresql_suffixes} {
+    set vrt postgresql${suffix}
+    set pgversion [string index ${suffix} 0].[string index ${suffix} 1]
+    set index [lsearch -exact ${postgresql_variants} ${vrt}]
+    set conf [lreplace ${postgresql_variants} ${index} ${index}]
+
+    variant ${vrt} description "Use PostgreSQL ${pgversion}" conflicts {*}${conf} "
+        depends_lib-append      port:${vrt}
+        configure.args-append   --with-postgres-includes=${prefix}/include/${vrt}
+        configure.args-append   --with-postgres-libs=${prefix}/lib/${vrt}
+        configure.args-delete   --with-postgres=no
+        configure.args-append   --with-postgres=yes
+    "
+}
+
+# PostGreSQL default
+set pgdefault "if {"
+
+foreach suffix ${postgresql_suffixes} {
+    set pgdefault "${pgdefault}!\[variant_isset postgresql${suffix}\] && "
+}
+
+set pgdefault [string range ${pgdefault} 0 end-4]
+set pgdefault "${pgdefault}} { default_variants +postgresql${suffix} }"
+
+eval ${pgdefault}
+
+variant sqlite description {Add SQLite 3 support} {
+    depends_lib-append    port:sqlite3
+    configure.args-append --with-sqlite
+    configure.args-append --with-sqlite-includes=${prefix}/include
+    configure.args-append --with-sqlite-libs=${prefix}/lib
+}
+
+variant openblas description {Use OpenBLAS for BLAS/LAPACK} {
+    depends_lib-append      port:openblas
+
+    configure.args-delete   --with-blas=no
+    configure.args-delete   --with-lapack=no
+    configure.args-append   --with-blas=yes
+    configure.args-append   --with-lapack=yes
+    configure.args-append   --with-blas-includes=${prefix}/include
+    configure.args-append   --with-libblas=${prefix}/lib/libopenblas.dylib
+    configure.args-append   --with-lapack-includes=${prefix}/include
+    configure.args-append   --with-liblapack=${prefix}/lib/libopenblas.dylib
+}
+
+post-configure {
+    reinplace -E "s|\\\$\\\(CPPFLAGS)| \$(CPPFLAGS) \
+        \\\\\"-Drestrict=\\\\\" \\\\\"-D__attribute__(x)=\\\\\" \\\\\"-D_Nonnull=\\\\\" \
+        \\\\\"-Dint8_t=char\\\\\" \\\\\"-DCF_INLINE=\\\\\" \\\\\"-D_Null_unspecified=\\\\\" \
+        \\\\\"-D__DARWIN_OS_INLINE=\\\\\"|" ${worksrcpath}/python/libgrass_interface_generator/Makefile
+}
+
+livecheck.type      regex
+livecheck.url       ${master_sites}
+livecheck.regex     grass-(8.\[0-9.\]+)${extract.suffix}
```

```suggestion
livecheck.regex     grass-(8\\.\[0-9.\]+)[quotemeta ${extract.suffix}]
```
However, this isn't the "grass8" port; this is the "grass" port. Why limit livecheck to version 8.x? Wouldn't we want to upgrade to 9.x when that becomes available?

---

Indeed, updated. Please check if I got the regex right (eg. `grass-X.Y.Z.tar.gz`).